### PR TITLE
[14.0][MIG] l10n_ch_base_bank

### PR DESCRIPTION
**Pull request by SimoneVagile**

### CODE REVIEW

File: l10n_ch_base_bank/tests/test_bank.py [hunk 1]
```diff
@@ -173,7 +181,7 @@ def test_iban_postal_lower_case(self):
         self.assertEqual(account.acc_type, "iban")
 
     def test_other_bank(self):
-        self.bank.l10n_ch_postal = False
+        # self.bank.l10n_ch_postal = False
```
Comment (yvaucher) [inline l10n_ch_base_bank/tests/test_bank.py]: You can remove this line.

This is an artefact of the former setUp function which was setting `ccp` on `self.bank`

https://github.com/OCA/l10n-switzerland/commit/118c9b2a71fb758dff2d7ab0c17f937420bd3199#diff-43f9985cce610ea14560cfe3362aa6162fbe069cb87560713657b3ae0758c170R22

File: l10n_ch_base_bank/tests/test_create_invoice.py [hunk 2]
```diff
@@ -84,17 +84,15 @@ def test_emit_move_with_missing_isr_ref(self):
...
-        inv_form.invoice_partner_bank_id = self.env["res.partner.bank"]
+        inv_form.partner_bank_id = self.env["res.partner.bank"]
         with self.assertRaises(exceptions.ValidationError):
             inv_form.ref = "132000000000000000000000014"
             inv_form.save()
 
     def test_emit_move_with_isr_ref_subscr_num_wrong_currency(self):
         inv_form = self.new_form()
         move = inv_form.save()
-        inv_form.save()
```
Comment (yvaucher) [inline l10n_ch_base_bank/tests/test_create_invoice.py]: nice catch

**Review by yvaucher — COMMENTED** [review]
You will also want to update the `__manifest__.py`

**Comment by yvaucher** [comment]
On a side note, this module has less and less purpose of being named `base_bank`. As now it mostly includes UX improvements.

**Comment by SimoneVagile** [comment]
> You will also want to update the `__manifest__.py`

:sweat_smile:

File: l10n_ch_base_bank/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -0,0 +1,8 @@
+* Nicolas Bessi (Camptocamp)
+* Vincent Renaville <vincent.renaville@camptocamp.com>
+* Joël Grand-Guillaume <joel.grandguillaume@camptocamp.com>
+* Guewen Baconnier <guewen.baconnier@camptocamp.com>
+* Paul Catinean <paulcatinean@gmail.com>
+* Yannick Vaucher <yannick.vaucher@camptocamp.com>
+* Akim Juillerat <akim.juillerat@camptocamp.com>
+* Iryna Vyshnevska <i.vyshnevska@mobilunity.com>
```
Comment (yvaucher) [inline l10n_ch_base_bank/readme/CONTRIBUTORS.rst:8]: You can add your name at the end of the file too.

Comment (SimoneVagile) [reply l10n_ch_base_bank/readme/CONTRIBUTORS.rst:8]: done

**Review by yvaucher — APPROVED** [review]

**Pull request merged**